src/main/java/at/hannibal2/skyhanni/features/fishing/FishingTimer.kt

```diff
@@ -67,6 +69,7 @@ class FishingTimer {
 
         if (inHollows && newCount >= 60 && config.wormLimitAlert) {
             SoundUtils.playBeepSound()
+            TitleManager.sendTitle("§cWORM CAP FULL!!!", 2.seconds, 7.2)
```

Comment:
Please change this to LorenzUtils.sendTitle()
if you want you can also change the other two instances of TitleManager.sendTitle() in the codebase to the LorenzUtils one